src/Pact/Types/Term.hs

```diff
@@ -1144,7 +1155,7 @@ termCodec = Codec enc dec
            , meta .= tmeta, inf i ]
       TDynamic r m i ->
         ob [ dynRef .= r, dynMem .= m, inf i ]
-      TModRef mr _i -> toJSON mr
+      TModRef mr i -> ob [ modRef .= mr, inf i ]
```

Comment:
This looks like this breaks DB compat. I think we have to leave `ModRef` alone and seek to fix this in the `PactValue` constructor.